rootfs/scheduler/tests/test_deployments.py

```diff
@@ -73,6 +76,47 @@ def scale(self, namespace=None, name=generate_random_name(), **kwargs):
         self.scheduler.scale(namespace, name, **kwargs)
         return name
 
+    def unsanitized_version(self):
+        return parse("1.9+")
+
+    def sanitized_version(self):
+        return parse("1.9")
+
+    def test_deployment_api_version_1_9_and_up(self):
+        expected = 'apps/v1'
+        deployment = copy.copy(self.scheduler.deployment)
+
+        deployment.version = mock.MagicMock(return_value=parse("1.9+"))
+        self.assertEqual(expected, deployment.api_version, '1.9+ breaks')
```

Comment:
What exactly is the meaning of the word "breaks" here?

This passed CI, so I'm assuming that actually nothing is breaking since all of the tests are green...

Reply:
Never mind, I get it.  assertEqual takes three arguments, and the third one is displayed if something goes wrong here.  Thanks!

Reply:
Indeed. However, I should remove those two "version" methods that I added to the test, as they aren't actually used, and I'd forgotten about them.